src/pages/Home.js

```diff
@@ -141,7 +153,7 @@ class Home extends React.Component {
                                                 this.state.search ? false : true
                                             }
                                         >
-                                            Buscar cápsula
+                                        {this.state.searchButton}
```

Comment:
💡😮 I think we could use one less state property simply by reusing `this.state.isLoadingData`, and we would get the expected result. In the content of the button it would be:
```javascript
this.state.isLoadingData ? 'Loading...' : 'Search Capsule'
```
Then it would simply be updated in place of the `searchButton` property to have the exact behavior of it.

Reply:
True, make the change.